.pre-commit-config.yaml

```diff
@@ -1,26 +1,22 @@
 repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: a11d9314b22d8f8c7556443875b731ef05965464
+    rev: v4.6.0
     hooks:
+    -   id: check-added-large-files
+        args: ['--maxkb=512']
```

Comment:
I'm more concerned that 512kb won't be enough for our use, as some of the previous files had more than 512kb in them.

Reply:
This will only be applied to local staged files as documented here: https://github.com/pre-commit/pre-commit-hooks?tab=readme-ov-file#check-added-large-files. which means, current .github/workflows/codestyle.yml will not capture large files being added.

Another option : increase the limit to 1024kb, and check all the files on each pull_request by using `--enforce-all`, at the same time, exclude existing large files. there are 28 files larger than 1M in the repo.